app/models/chat_message.rb

```diff
@@ -27,6 +27,10 @@ def excerpt
     PrettyText.excerpt(cooked, 50, {})
   end
 
+  def push_notification_excerpt
+    message[0...400]
```

Comment:
I think this is slightly better, we have still no idea why 400 tho. Given you've tried your best to explain but can't yet find anything better, let's let this go and please revisit if you suddenly 💡 

(Is it a char LIMIT?)